### DISCUSSION TITLE
Off Topic: Convert chromeOS into gentoo?

### DISCUSSION THREAD

**Discussion by cdd100** [dnschneid/crouton#4848] [Q&A] [answered]
How could i possibly convert chrome os into normal gentoo with a gdm3 or lightdm login screen on boot and a normal linux desktop enviroment?

**Comment by TomTravis** [comment]
chomeOS as a chroot is this what you thinking, maybe 
not sure the idea is google idea so like is not made to do this
google dev
https://chromium.googlesource.com/chromiumos/docs/+/HEAD/developer_guide.md

**Comment by cdd100** [comment]
Not exactly, I mean to revert all the changes to gentoo that google made to get stock gentoo linux.

**Reply by TomTravis** [reply]
what did google do? just load the org kernel and have fun
https://www.kernel.org/

**Comment by DennisLfromGA** [comment]
@cdd100,

If your device is [supported by mr.chromebox](https://wiki.mrchromebox.tech/Supported_Devices) you could install his custom UEFI firmware and then install [the Gentoo distro](https://www.gentoo.org/downloads/).

-**DennisLfromGA**

**Reply by cdd100** [reply]
I have an ARM chromebook, so I will probably just stick to chromeOS.

**Comment by DennisLfromGA** [comment, marked as answer]
@cdd100,

Sounds like a good plan.
You could buy a reasonably priced [Linux laptop](https://www.howtogeek.com/748445/best-linux-laptops/) as an alternative.

*Hope this helps,*
-**DennisLfromGA**

**Reply by cdd100** [reply]
Might give github.com/hexdump0815/imagebuilder a try since I have a pc but it is probably best to stay on chrome os.

**Comment by lucasgabmoreno** [comment]
> How could i possibly convert chrome os into normal gentoo with a gdm3 or lightdm login screen on boot and a normal linux desktop enviroment?

Install Gentoo into ChromeBook: https://wiki.gentoo.org/wiki/Chromebook
ARM install: https://wiki.gentoo.org/wiki/ASUS_Chromebook_C201/Installing_Gentoo
